memoized_lcsh.txt

```diff
@@ -0,0 +1 @@
+{"Sailboats"=>"sh2011004125", "Periodical cicada"=>"sh85099887", "Saguenay River (Quebec)"=>"sh86006143", "Social stratification"=>"sh99000315", "Originality"=>"sh85095646", "Sailing"=>"sh91001077", "Marsden"=>"sh99014562", " Sam (Fictitious character)"=>"sh2008000533", "Molds (Fungi)"=>"sh85086566", "Lindens"=>"sh85077148", "Statues"=>"sh2003001401", "Test family"=>"sh85080045", "Semantic Web"=>"sh2002000569"}
```

Comment:
This and the `memoized_mesh.txt` results were from staging? Surprisingly small results to me. I expect production will be much larger. When we export on production, let's remember to get those results and commit it into the repository.

Reply:
Yeah, I was surprised too. I might run a little query to check myself on this, but I could see this being legit given it's basically a testing environment. If it seems like prod is isn't getting very many results we might need to dig deeper on this.